packages/webcert/src/feature/certificate/PatientAddress/PatientAddress.tsx

```diff
@@ -0,0 +1,86 @@
+import { TextInput } from '@frontend/common'
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components/macro'
+import { getPatient } from '../../../store/certificate/certificateSelectors'
+import CategoryHeader from '../Category/CategoryHeader'
+import CategoryTitle from '../Category/CategoryTitle'
+import QuestionWrapper from '../Question/QuestionWrapper'
+
+const Wrapper = styled.div`
+  align-items: center;
+`
+
+const ZipCodeInput = styled(TextInput)`
+  max-width: 6.5em;
```

Comment:
Could be a tiny bit larger to allow all characters to show on different systems

![image](https://user-images.githubusercontent.com/28404/196894983-74937f26-ac81-4a0b-8fb0-b8ec7054f4f8.png)

Reply:
It has the same measurements as the field for zip code in the component for care unit address. The field will probably only hold Swedish zip codes so I will leave it this size for now. :)